schemas/sml_links.graphql

```diff
@@ -0,0 +1,60 @@
+### Types
```

Comment:
The file should probably be renamed to `sml_archs.graphql` and all objects should follow the same convention as per the rename.

Reply:
https://github.com/satisfactorymodding/smr-api/pull/12/commits/bdd565d8997568955b8e33bcb352c38c7af5c2c5